.circleci/config.yml

```diff
@@ -24,11 +24,10 @@ jobs:
           - py3-cache-v9-{{ arch }}-{{ .Environment.BASE_BRANCH }}
       - run:
           name: Install python dependencies
-          # FIXME: For now, we fix setuptools due to https://github.com/etalab/data.gouv.fr/issues/1041
           command: |
-            virtualenv venv
+            python -m venv venv
             source venv/bin/activate
-            pip install --force-reinstall setuptools==66.1.1
+            pip install --upgrade wheel
```

Comment:
Not sure about the need to install wheel explicitly. See https://stackoverflow.com/questions/69369669/python-3-venv-and-the-wheel-package